blueocean-personalization/src/main/js/components/PipelineCard.jsx

```diff
@@ -96,7 +96,7 @@ export class PipelineCard extends Component {
                 { this.props.branch ?
                 <span className="branch">
                     <span className="octicon octicon-git-branch"></span>
-                    <span className="branchText">{this.props.branch}</span>
+                    <span className="branchText">{decodeURIComponent(this.props.branch)}</span>
```

Comment:
Would be nice if this was an object and there was a `branch.name` property for display